src/python/TaskWorker/Worker.py

```diff
@@ -54,8 +52,8 @@ def failTask(taskName, crabserver, msg, log, failstatus='FAILED'):
         configreq = {'workflow': taskName,
                      'status': failstatus,
                      'subresource': 'failure',
-                     # Limit the message to 7500 chars, which means no more than 10000 once encoded. That's the limit in the REST
-                     'failure': b64encode(encodeUnicodeToBytes(truncMsg))}
+                     # Limit the message to 10000. That's the limit in the REST
```

Comment:
@mapellidario why did you increase this ? Maybe the previous limit had a rational (even if not documented clearly)

Reply:
From the comment that I removed, I get that there is some in the REST, at aroun 10k characters [1]. Since base64 on average consumes 133% space of the original string [2], in the past we restricted to 7.5k characters before encoding, so that after encoding they would have benn around 10k. 

Since we do not b64-encode anymore, we can increase the size of the unicode string to 10k characters without hitting any limit.

[1] actually i tried and managed to succesfully upload a warning containing 100k characters to `220311_094223:dmapelli_crab_20220311_104220`, so i am not sure that comment what is based on. the column that holds the warning messages is a `CLOB` and it [seems](https://docs.oracle.com/cd/B19306_01/server.102/b14237/limits001.htm#i287903) that can contain 4GB of data. Maybe the limitation comes from the old version of cherrypy.

[2]: https://developer.mozilla.org/en-US/docs/Glossary/Base64#encoded_size_increase

Reply:
I think the limit at 7500 comes from https://github.com/dmwm/CRABServer/blob/d6fe22eeab537c5eceaf583b3df8ef744405a9f5/src/python/ServerUtilities.py#L159-L169
(followoing up the GH issue indicated in the comments in the code does not add any light, at some point the 7500 limit in the REST was hit and people decided to truncate client side).
I am not sure why that particular value was picked (maybe initially they were using some VCARCHAR instead of CLOB in Oracle ?), but surely messages that long make little sense.
So it is simply a matter of being consistent. We can go for 10k chars, but then let's do it uniformly,
simply to avoid headache to future code maintainers.

Reply:
> maybe initially they were using some VCARCHAR instead of CLOB in Oracle

Without any investigation, i think i agree.

> but surely messages that long make little sense.

I agree with this, too.

> So it is simply a matter of being consistent

I should have changed `ServerUtilities.py` in this PR, raising the limit to 10k characters.

![immagine](https://user-images.githubusercontent.com/16376695/158078966-281f7370-fc2e-47d4-8181-8f5b97689b9c.png)

Is this what you mean in " it is simply a matter of being consistent."? Or do you suggest I change something esle, too?